app/models/work_package/pdf_export/work_package_list_to_pdf.rb

```diff
@@ -294,10 +319,15 @@ def make_custom_field_value(work_package, column)
                   padding: cell_padding
   end
 
-  def initialize_batch_page
-    self.pdf = get_pdf(current_language)
+  def batch_supported?
+    return @batch_supported if defined?(@batch_supported)
 
-    configure_page_size
-    configure_markup
+    @batch_supported = begin
+      _, status = Open3.capture2e('pdfunite', '-h')
```

Comment:
We test this already in the admin info view, don't we? Would be good not to duplicate the code for that. Maybe we can put this into a module which can be used in both spots? Ideally with caching of the result so that the actual system call only has to be done once.

Reply:
We only test for poppler-utils afaik, and you can have `pdfunite` available independently from it (however installing poppler-utils will install it as well)